Sources/App/Views/PackageController/PackageShow+StructuredDataSchema.swift

```diff
@@ -20,7 +20,7 @@ extension PackageShow {
         enum CodingKeys: String, CodingKey {
             case context = "@context", type = "@type"
             case identifier, name, description, license, version,
-                 codeRepository, url, dateCreated, dateModified,
+                 codeRepository, url, dateCreated = "datePublished", dateModified,
```

Comment:
I think we should update the underlying attribute to `datePublished` instead of rewriting the key here. We're already mapping once from `firstCommitDate` → `dateCreated` and then encoding that as `datePublished` will get confusing in the future :)